views/account/settings/index.ejs

```diff
@@ -9,41 +9,144 @@
 
-    <div
-      class="cover-container w-100 d-flex align-items-center p-3 mx-auto"
-    >
-      <div class="row col-12">
-        <main class="px-3">
-          <h1 class="display-4">Goon up Brah.</h1>
-          <h1>Hello World!</h1>
-          <p>This file was served with Express.</p>
-          <p>Here is your information:</p>
-          <ul>
-            <li>First Name: <%= user.first_name %></li>
-            <li>Last Name: <%= user.last_name %></li>
-            <li>Email: <%= user.email %></li>
-            <li>Password: <%= user.password %></li>
-            <li>
-              Settings:
-              <ul>
-                <% var settings = JSON.parse(user.settings);
-                Object.keys(settings).forEach( (key) => { %>
-                <li><%= key %>: <%= settings[key] %></li>
-                <% }); %>
-              </ul>
-            </li>
-          </ul>
-
-          <a
-            href="#"
-            class="btn btn-warning btn-lg"
-            role="button"
-            aria-disabled="true"
-            >Generate</a
-          >
-        </main>
-      </div>
+    <!-- Parse the user's settings as JSON before using. -->
+    <% var settings = JSON.parse(user.settings); %>
+
+    <!-- Create the options. -->
+    <%var options = { font_size: ["12pt", "14pt", "16pt", "18pt"], font_family:
+    ["Times New Roman", "Arial", "Helvetica", "Sans-Serif"], contrast: ["Low",
+    "High"], theme: ["Light Theme", "Dark Theme"], }; %>
+
+    <div class="col-sm-2">
+      <br />
+      <h1>Settings</h1>
+      <p class="message text-danger"><%= error %></p>
     </div>
+
+    <br />
+    <form action="/account/settings/update" method="POST">
+      <div class="form-group row">
+        <label for="editFirstName" class="col-sm-2 col-form-label"
+          >First Name:</label
+        >
+        <div class="col-sm-2">
+          <input
+            name="first_name"
+            type="firstName"
+            class="form-control"
+            id="editFirstName"
+            value="<%= user.first_name %>"
+          />
+        </div>
+      </div>
+
+      <br />
+      <div class="form-group row">
+        <label for="editLastName" class="col-sm-2 col-form-label"
+          >Last Name:</label
+        >
+        <div class="col-sm-2">
+          <input
+            name="last_name"
+            type="lastName"
+            class="form-control"
+            id="editLastName"
+            value="<%= user.last_name %>"
+          />
+        </div>
+      </div>
+
+      <br />
+
+      <div class="form-group row">
+        <label for="inputFontSize" class="col-sm-2 col-form-label"
+          >Font Size:</label
+        >
+        <div class="col-sm-2">
+          <div class="form-group">
+            <select name="font_size" class="form-control" id="inputFontSize">
+              <!-- Adds the options and selects the user's current setting. -->
+              <% options.font_size.forEach( (option) => { if (settings.font_size
```

Comment:
I know that this way is a bit tedious, but EJS allows us to use values directly without having to make a separate request in the main.js file. I'm creating an issue that looks into how complex making the change would be.